src/patternfly/utilities/Spacing/spacing.hbs

```diff
@@ -1,2 +1,5 @@
-<div class="item{{#if spacing--modifier}} {{spacing--modifier}}{{/if}}">
-</div>
+<div class="item{{#if spacing--modifier}} {{spacing--modifier}}{{/if}}"
+{{#if spacing--attribute}}
+    {{{spacing--attribute}}}
+{{/if}}>
```

Comment:
should there be a `{{> @partial-block}}` block here?

Reply:
yep, adding that in, thanks!